.github/workflows/build-suite-native-preview.yml

```diff
@@ -0,0 +1,59 @@
+name: "[Build] suite-native preview"
+
+on:
+  push:
+    # push develop(default) branch is necessary for this action to update its fingerprint database
+    branches: [develop]
+  pull_request:
+    paths:
+      - "suite-native/**"
+      - "packages/react-native-usb/**"
+      - "packages/transport-native/**"
+      - "yarn.lock"
+      # list of paths is not complete, but it's always possible to dispatch manually
+  workflow_dispatch:
+
+jobs:
+  update:
+    if: github.repository == 'trezor/trezor-suite' || github.repository == 'trezor/trezor-suite-private'
+    name: EAS Update
+    runs-on: ubuntu-latest
+    concurrency: fingerprint-${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      EXPO_PUBLIC_ENVIRONMENT: preview
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+    permissions:
+      contents: read
+      pull-requests: write
+      actions: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          cache: yarn
+      - name: Setup EAS
+        uses: expo/expo-github-action@v8
+        with:
+          eas-version: latest
+          token: ${{ secrets.EXPO_TOKEN_DEVELOP }}
+      - name: Install libs
+        run: yarn workspaces focus @suite-native/app
+
+      - name: Create preview builds if fingerprint changed
+        uses: expo/expo-github-action/preview-build@main
+        with:
+          command: eas build --profile preview --platform all
+          working-directory: suite-native/app
+
+      - name: Install deps and build libs
```

Comment:
Why are you calling `yarn install` again here? `yarn workspaces focus @suite-native/app` above should allready have installed all dependendecies needed

Reply:
👍 I wasn't sure all the dependencies for message system config are there, but it works fine, so removed, thanks for noticing!